### ISSUE TITLE
ublk del --all always returns failure

### ISSUE THREAD

**Issue by vt-alt** [ublk-org/ublksrv#61]
I did simple smoke test for our CI and found that `ublk del --all` always exits with a failure (exitcode=237).

Looking at the code I think this is the casue:

`cmd_dev_del` iterates over `MAX_NR_UBLK_DEVS` number of devices:
```
        for (i = 0; i < MAX_NR_UBLK_DEVS; i++)
                ret = __cmd_dev_del(i, true);
```
1. This overrides exit status (of intermediate dels), and
2. Always reporting exit for last (128th) device (which is usually non-existent), which is `-19` (that becomes exitcode 237).

**Comment by ming1** [comment]
Fixed in bac8d72 ("ublksrv_tgt: don't return error if trying to delete unexisted device").

Thanks for the report.

**Comment by vt-alt** [comment]
Thanks!
